README.md

```diff
@@ -3,3 +3,40 @@
 An open source, cleanroom recreation of [Extra Utilities](https://www.curseforge.com/minecraft/mc-mods/extra-utilities) for 1.7.10.
 
 Optionally dependent on [Angelica](https://github.com/GTNewHorizons/Angelica) for connected texture mapping.
+
+🚨 OFFICIAL DISCLAIMER 🚨
+
+In case of an investigation 🔍 by any federal 🏛️, corporate 🏢, magical 🧙‍♂️, or interdimensional 🌌 entity, I hereby state the following:
+
+I have NOT ❌
+- read the source code 📜
+- glanced at the source code 👀
+- skimmed the source code 🏃‍♂️
+- reverse engineered the source code 🧠
+- thought about the source code 💭
+- dreamed about the source code 😴
+- or acknowledged the existence of the source code 🫥
+
+of Extra Utilities in ANY way, shape, or form.
+
+My work on Utilities In Excess (aka the ✨ CLEANROOM ✨ rewrite) is based solely on:
+✔️ vibes
+✔️ documented behavior 📚
+✔️ public knowledge 🌍
+✔️ divine inspiration ⚡
+✔️ and sheer willpower 💪
```

Comment:
i think this should be either
```suggestion
- ✔️ vibes
- ✔️ documented behavior 📚
- ✔️ public knowledge 🌍
- ✔️ divine inspiration ⚡
- ✔️ and sheer willpower 💪
```
or perhaps
```suggestion
<br>✔️ vibes
<br>✔️ documented behavior 📚
<br>✔️ public knowledge 🌍
<br>✔️ divine inspiration ⚡
<br>✔️ and sheer willpower 💪
```
so it looks like a list instead of 

<img width="1034" height="558" alt="Image" src="https://github.com/user-attachments/assets/d537c5cf-f9e8-4bf8-a1bd-5cb8379c8ed0" />